src/app/news.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Http, Jsonp } from '@angular/http';
+import 'rxjs/add/operator/map';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class NewsService {
+
+  public sportsNews: string = 'https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=b487d01b4a044b11aba9e33c77850963';
```

Comment:
The idea of our news service is not to use APIs of some news services at this point. We simply want a filter of certain search results. Please read the issue carefully again!